docs/support.md

```diff
@@ -0,0 +1,9 @@
+# Getting Support
+
+For issues relating to documentation, facility deployments, and requests to include new packages in E4S, please file an issue here:
+* https://github.com/E4S-Project/e4s/issues
+
+For all other issues, please feel free to raise them in one of the following ways:
+1. [E4S Project Slack](https://e4s-project.slack.com)
+2. [E4S Channel on Spack Slack](https://spackpm.slack.com)
+3. Email: sameer AT cs.uoregon.edu
```

Comment:
better way would be do `[sameer@cs.uoregon.edu](mailto:sameer@cs.uoregon.edu)` that way when you click the link it will open up the email

click on this link --> [sameer@cs.uoregon.edu](mailto:sameer@cs.uoregon.edu)

Reply:
done